Render/Render.as

```diff
@@ -257,10 +285,7 @@ void RenderTab(bool showRefresh = false){
                     break;
             }
         } else if(leaderboardArray[i].entryType == EnumLeaderboardEntryType::PB){
-            // if we're in an early update state, grey out the pb
-            if(updatePbEarlySetting && earlyPBUpdate){
-                displayString = greyColor;
-            }
+            displayString = greenColor;
```

Comment:
May I suggest having the PB being green being an option? (that can be true by default tho)

Reply:
Certainly (I had it in there to look like the Nadeo leaderboard). Should it be like the medals dropdown with multiple colors (normal/gray/green I guess)? Or maybe just checkbox under Show personal best for now, and wait for a future color feature overhaul for any additional options?

Reply:
A dropdown would be nice, since the color overhaul is a big soon:tm:, as in idk when I'll work on it x), maybe in March, but nothing's sure 😅